mffpy/writer.py

```diff
@@ -84,21 +84,21 @@ def addxml(self, xmltype, filename=None, **kwargs):
     def addbin(self, binfile: BinWriter, filename=None):
         """Add the .bin file to the collection
 
-        Currently we only allow to add one such files, b/c .mff can only have
-        one `epochs` file.  For this we added the flag `self._bin_file_added`.
-
         **Parameters**
 
         *binfile*: `class BinWriter` to be added to the collection
-        *filename*: (defaults to `binfile.default_filename`) filename of the
-            bin file.  It's not recommended to change this default value.
+
+        *filename*: (defaults to `binfile.default_filename_fmt %
+            self.num_bin_files`) filename of the bin file.  It's not
+            recommended to change this default value.
         """
-        assert not self._bin_file_added
-        self.files[filename or binfile.default_filename] = (
-            binfile, type(binfile))
-        self.addxml('dataInfo', **binfile.get_info_kwargs())
+        self.num_bin_files += 1
+        binname = filename or \
+            (binfile.default_filename_fmt % self.num_bin_files)
+        infoname = binfile.default_info_filename_fmt % self.num_bin_files
+        self.files[binname] = (binfile, type(binfile))
+        self.addxml('dataInfo', filename=infoname, **binfile.get_info_kwargs())
         self.addxml('epochs', epochs=binfile.epochs)
```

Comment:
I think we will end up trying to add multiple `epochs.xml` files here if we use this `Writer.addbin` function more than once. Maybe check to see if an `epochs.xml` has been added and if not, then add one?